ReadMe.md

```diff
@@ -111,3 +111,36 @@ Please check the contributing guidelines of my previous project, and it's going
+<td align="center"><a href="https://github.com/TeeVenDick"><img src="https://avatars.githubusercontent.com/u/56723862?s=400&v=4" width="100px;" alt=""/><br /><sub><b>TeeVenDick</b></sub></a><br /><a href="https://github.com/TeeVenDick/Internship-LMS-FrontEnd/commits?author=TeeVenDick" title="Code">💻</a></td>
+<td align="center"><a href="https://github.com/muthuannamalai12"><img src="https://avatars.githubusercontent.com/u/64524822?s=400&u=c1f8f317ca1eb1340f411b69b3b7c85446303ae5&v=4" width="100px;" alt=""/><br /><sub><b>muthuannamalai12</b></sub></a><br /><a href="https://github.com/muthuannamalai12/Internship-LMS-FrontEnd/commits?author=muthuannamalai12" title="Code">💻</a></td>
+<td align="center"><a href="https://github.com/Jassi10000"><img src="https://avatars.githubusercontent.com/u/64856348?s=400&u=aa16e7590ad90c40bb1a2a6a987fb78dbd634237&v=4" width="100px;" alt=""/><br /><sub><b>Jassi10000</b></sub></a><br /><a href="https://github.com/Jassi10000/Internship-LMS-FrontEnd/commits?author=Jassi10000" title="Code">💻</a></td>
+<td align="center"><a href="https://github.com/tharunc"><img src="https://avatars.githubusercontent.com/u/68283386?s=400&v=4" width="100px;" alt=""/><br /><sub><b>tharunc
+</b></sub></a><br /><a href="https://github.com/tharunc/Internship-LMS-FrontEnd/commits?author=tharunc" title="Code">💻</a></td>
+<td align="center"><a href="https://github.com/Ayush7614"><img src="https://avatars.githubusercontent.com/u/67006255?s=400&u=c0e16c3bba31328a028cfcca4b1fa7599509f905&v=4" width="100px;" alt=""/><br /><sub><b>Ayush7614</b></sub></a><br /><a href="https://github.com/Ayush7614/Internship-LMS-FrontEnd/commits?author=Ayush7614" title="Code">💻</a></td>
+  </tr>
+</table>
+
+
+<h2 align=center> Stargazers Over Time 🌟 </h2>
+
+[![Stargazers over time](https://starchart.cc/praveenscience/Internship-LMS-FrontEnd.svg)](https://starchart.cc/praveenscience/Internship-LMS-FrontEnd)
+
+
+## ❤️ Project Admin
+
+|                                     <a href="https://github.com/praveenscience"><img src="https://avatars.githubusercontent.com/u/1830380?s=400&u=74697d8b1cbf3e16adec7b411369afbd53ce4864&v=4" width=150px height=150px /></a>                                      |
+| :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+|                                                                                      **[Praveen Kumar Purushothaman](https://www.linkedin.com/in/praveentech/?originalSubdomain=uk)**                                                                                       |
```

Comment:
Looks really good, but can we please get rid of the HTML here? Use Markdown. 😁

Reply:
> Looks really good, but can we please get rid of the HTML here? Use Markdown. 😁

I use less HTML from my side Now but Without HTML it is impossible so plss understand sir because even if i not use HTML then the name of contributers is not in the sequence or not in line by line